### PR TITLE
Quadrat: add custom logo support

### PR DESCRIPTION
**Pull request by jffng**
This PR adds support for a custom logo. It depends on this PR: https://github.com/WordPress/gutenberg/pull/31513 

Screen Shot 2021-05-05 at 12 06 11 PM

This is a part of #3661

### CODE REVIEW

File: quadrat/template-parts/header.php [hunk 1]
```diff
@@ -14,8 +14,11 @@
 <header class="site-header" role="banner">
 	<?php
 	echo do_blocks(
-		'<!-- wp:group {"style":{"spacing":{"padding":{"right":"35px","left":"35px"}}}} --><div class="wp-block-group" style="padding-right:35px;padding-left:35px">
-		<!-- wp:site-title /-->' .
+		'<!-- wp:group {"style":{"spacing":{"padding":{"top":"35px","right":"35px","left":"35px"}}}} --><div class="wp-block-group" style="padding-top:35px;padding-right:35px;padding-left:35px">
+		<!-- wp:group --><div class="wp-block-group">
+		<!-- wp:site-logo {"width":100} /-->
```
Comment (jffng) [inline quadrat/template-parts/header.php:19]: Not sure what the default width should be.

**Comment by kjellr** [comment]
We'll need to adjust the alignment + location of the site title and nav when this logo is present. @beafialho can you provide a recommendation?

**Comment by beafialho** [comment]
Logo's default width should be 150px

On Desktop, it should be placed like this, and the site title and nav should be middle aligned with the logo.

Captura de ecrã 2021-05-06, às 10 56 42

Captura de ecrã 2021-05-06, às 10 57 47

On Mobile, it's placed like this and the site title and nav are middle aligned below the logo.

Captura de ecrã 2021-05-06, às 10 50 18

**Comment by jffng** [comment]
Closing as it's a duplicate, but will still reference the design mocks here.

**Pull request closed without merging**